h/templates/includes/annotation_card.html.jinja2

```diff
@@ -62,7 +62,14 @@
   <footer class="annotation-card__footer">
     {% if presented_annotation.incontext_link %}
       <a href="{{ presented_annotation.incontext_link }}" target="_blank" title="{% trans %}Visit annotation in context{% endtrans %}">
-        {{ svg_icon('up-right-arrow', 'annotation-card__incontext-link') }}
+        {{ svg_icon('up-right-arrow', 'annotation-card__footer-link annotation-card__incontext-link') }}
+      </a>
+      <a href="#" share-widget-config='{
```

Comment:
I haven't closely looked at keyboard or screen-reader accessibility for this, and it isn't too much of an issue since the incontext link is available as a plain link right text next to it. However, I think there are a couple of basic things we can at least do:

- Set a `title` attribute on the link
- Set the `aria-haspopup="true"` attribute on the link